assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/profiles.tf

```diff
@@ -76,15 +74,13 @@ resource "matchbox_profile" "cached-container-linux-install" {
     var.kernel_args,
   ])
 
-  container_linux_config = data.template_file.cached-container-linux-install-configs[count.index].rendered
+  container_linux_config = data.ct_config.cached-container-linux-install-configs[count.index].rendered
 }
 
-data "template_file" "cached-container-linux-install-configs" {
+data "ct_config" "cached-container-linux-install-configs" {
```

Comment:
Again, `ct_config` shouldn't be used here.

Reply:
Isn't this os related?

Reply:
Ah yes, my bad, sorry :pray: